pkg/build/api/types.go

```diff
@@ -19,9 +19,6 @@ type Build struct {
 	// A human readable message indicating details about why the build has this status
 	Message string `json:"message,omitempty"`
 
-	// PodName is the name of the pod that is used to execute the build
-	PodName string `json:"podName,omitempty"`
```

Comment:
This issue #408, but since I've added copying labels from Build to Pod, I'll also add Build.Name in Pod's labels as well (see 3rd commit). I hope that addresses your concerns.